x-pack/legacy/plugins/apm/public/components/shared/Summary/ErrorCountSummaryItem.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import styled from 'styled-components';
+import { ErrorCountBadge } from '../../app/TransactionDetails/WaterfallWithSummmary/ErrorCountBadge';
+
+interface Props {
+  count: number;
+}
+
+const Badge = styled(ErrorCountBadge)`
+  margin: 3px 0;
```

Comment:
Using arbitrary units feels wrong.

Reply:
This is the only way I can get the error badge to align with the other badges. I think that's because the others are wrapped in a tooltip which adds a padding/margin one way or the other. Do you have any suggestions?

Reply:
If I'm getting nitpicky; Even with that added margin, the error badge is higher up than the other badges.

<img width="559" alt="Screenshot 2019-09-25 at 14 52 08" src="https://user-images.githubusercontent.com/4104278/65602458-39de2080-dfa4-11e9-8e63-f2ad362a79dc.png">

Reply:
I think aligning it vertically (with flexbox or similar) is usually better than doing it manually based on pixels.

Reply:
I've now opted for `px(units.eighth)`. I tried centering it vertically but I couldn't get it to work in a reasonable amount of time.